source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -114,7 +115,7 @@ We have committed to several organizational renames that will allow us greater f
 Simulation
 ----------
 
-- No new changes
+- Uses multi-file JSON save formats instead of INI and supports loading/saving the workspace; when started from the VS Code tool menu, these default to using the project directory for saving rather than the system-global location
```

Comment:
```suggestion
- Uses multi-file JSON save formats instead of ``ini`` and supports loading/saving the workspace; when started from the VS Code tool menu, these default to using the project directory for saving rather than the system-global location
```

Reply:
Why do ini and not json

Reply:
didn't notice the json. both should be done